src/components/breadcrumbs/breadcrumbs.style.ts

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+const StyledBreadcrumbs = styled.nav`
```

Comment:
```
const StyledBreadcrumbs = styled.nav`
  ol {
    list-style: none;
    display: flex;
    flex-wrap: wrap;
  }
`;
```

Reply:
You need to apply the styling to the `ol` now and remove the list-style etc

Reply:
```
const StyledBreadcrumbs = styled.nav`
  ol {
    list-style: none;
    display: flex;
    flex-wrap: wrap;
  }
`;
```
 
Is all you need here, the flex and `flex-wrap` on the `nav` don't do anything here